options/posix/generic/syslog-stubs.cpp

```diff
@@ -20,8 +32,18 @@ void closelog(void) {
 
 static void __openlog() {
 	log_fd = socket(AF_UNIX, SOCK_DGRAM | SOCK_CLOEXEC, 0);
-	if(log_fd >= 0)
-		connect(log_fd, (const sockaddr *)&log_addr, sizeof log_addr);
+	if(log_fd >= 0) {
+		int ret = connect(log_fd, (const sockaddr *)&log_addr, sizeof log_addr);
+		if(ret) {
+			if(errno == ENOENT) {
+				mlibc::infoLogger() << "\e[31mmlibc: connect returned ENOENT, falling back to infoLogger\e[39m" << frg::endlog;
+				use_mlibc_logger = 1;
+			} else {
```

Comment:
Why the different codepaths here? is reporting ENOENT explicitely that important?